.devcontainer/Dockerfile

```diff
@@ -3,10 +3,10 @@ FROM ruby:3.2.1
 # install dependencies
 RUN apt-get update -qq && apt-get install -y imagemagick shared-mime-info libvips && apt-get clean
 
-RUN curl -sL https://deb.nodesource.com/setup_14.x | bash \
+RUN curl -sL https://deb.nodesource.com/setup_16.x | bash \
```

Comment:
Could you move this change to another PR ?

Reply:
sure

Reply:
done